sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/InternalBlobLegalHoldResult.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.implementation.models;
```

Comment:
I have a concern with putting handwritten models in a package where generated code lives. Any thoughts about that? @kasobol-msft

Reply:
ideally we'd have com.azure.storage.blob.implementation.generated.* for everything generated.
or have "hadwriten.models" :-). For this PR we can mix them, but it would be good to sit down and do some cleanup in implementation pacakge.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/22391